vllm/worker/worker_base.py

```diff
@@ -23,6 +23,7 @@ class WorkerBase(ABC):
     different hardware. Also abstracts control plane communication, e.g., to
     communicate request metadata to other workers.
     """
+    use_spmd_worker: bool
```

Comment:
let's make it a property + abstract method instead so that we can enforce setting this value from a child class?

Reply:
This is dynamically/lazily initialized: https://github.com/vllm-project/vllm/pull/6032/files#diff-1ca8953d4dab62fff08a34c8cf370d2a37aeb009526ab4fc5464a65e1d03b036R377
I think in this case there is not much benefits forcing child classes to implement the method (since the flag needs to be updated anyway)?